src/iop/cacorrect.c

```diff
@@ -385,7 +385,7 @@ void process(
 #pragma omp parallel default(none) \
   dt_omp_firstprivate(in, out, height, width, filters, processpasstwo, hblsz, vblsz, \
                       blockwt, blockshifts, blockvar, Gtmp, RawDataTmp, caautostrength, \
-                      border, border2, eps, eps2, v1, v2, v3, v4, ts, tsh) \
+                      border, border2, eps, eps2, v1, v2, v3, v4, ts, tsh, darktable) \
```

Comment:
Can you comment about this change? I don't see why... Maybe a left over of some debug?

Reply:
without it, gcc gives
```
[build] /home/diederik/darktable/src/iop/cacorrect.c: In function ‘process’:
[build] /home/diederik/darktable/src/common/darktable.h:416:42: error: ‘darktable’ not specified in enclosing ‘parallel’
[build]   416 |         || ((thread) & ~DT_DEBUG_VERBOSE &  darktable.unmuted && \
[build]       |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
[build] /home/diederik/darktable/src/common/darktable.h:421:31: note: in expansion of macro ‘dt_debug_if’
[build]   421 | #define dt_print(thread, ...) dt_debug_if(thread, dt_print_ext, __VA_ARGS__)
[build]       |                               ^~~~~~~~~~~
[build] /home/diederik/darktable/src/iop/cacorrect.c:755:15: note: in expansion of macro ‘dt_print’
[build]   755 |               dt_print(DT_DEBUG_PIPE, "[cacorrect] blockdenom vanishes\n");
```
llvm is perfectly happy with it though.

You may have noticed the same fix in #15552

Reply:
Oh I see because far down in the omp block we have a `dt_print` which is now using `darktable`!

Not easy to understand while reviewing a diff here.